practices/follow-functional-core-imperative-shell.md

```diff
@@ -3,117 +3,153 @@
 
-The Follow Functional Core, Imperative Shell practice significantly supports the Code Maintainability capability.
-By separating business logic into a functional core and side effects into an imperative shell, this approach enhances code readability, simplifies comprehension, and reduces complexity.
-With a clear distinction between pure functions and imperative code, developers can more easily understand and modify code, leading to improved maintainability and stability of the software system.
 
-### [Test Automation](/capabilities/test-automation.md)
+### [Facilitate a Roundtable Discussion](/practices/host-a-roundtable-discussion.md)
 
-The Follow Functional Core, Imperative Shell practice supports the Test Automation capability by facilitating the creation of highly testable and maintainable code. The functional core, being side-effect-free, allows for straightforward unit testing, as its pure functions yield predictable results and don't rely on external states. This ensures that the core business logic is thoroughly tested and reliable. The imperative shell handles side effects and interactions with external systems, which can be tested through integration tests. This clear separation simplifies the testing process, improves test coverage, and provides faster, more reliable feedback during development, which is crucial for robust and efficient test automation.
+Below are suggestions for topics and prompts you could explore with your team during a roundtable discussion. 
 
-### [Host A Roundtable Discussion](/practices/host-a-roundtable-discussion.md)
+<ed> Line above OK? Tweak as you like, but I recommend adding a quick sentence here to give context to the subsections and questions that follow. </ed>
 
 #### Assessing the Benefits
 
 * What advantages could the adoption of the Functional Core, Imperative Shell practice bring to our projects?
-* How might separating business logic from side effects enhance code readability, maintainability, and scalability?
-* In what ways could the Functional Core and Imperative Shell mitigate the impact of changes in infrastructure technology, allowing for smoother transitions and future-proofing our codebase?
-* What benefits might arise from writing unit tests with the assurance that the Functional Core code being tested has no side effects?
-* How could the reduced presence of control statements in the Imperative Shell simplify integration tests?
+* How might separating business logic from I/O enhance code readability, maintainability, and scalability? <ed> Edited "side effects" to "I/O". OK? </ed> 
```

Comment:
Is the goal to separate business logic from "side effects" or to separate business logic fro "I/O"?

Reply:
I/O can be considered a side effect. So let's just use the term side effect

Reply:
Will do!